src/storage-client/src/storage_collections.rs

```diff
@@ -1706,19 +1683,6 @@ where
         Ok(())
     }
 
-    fn drop_collections(
```

Comment:
This is currently dead code because `StorageController` wraps a `StorageCollections` and no-one uses `StorageCollections` directly. Eventually we'll have to call this.

Might be ok to remove and then re-add again when needed, though, because it at least confused you. 👌